scripts.py

```diff
@@ -569,11 +569,14 @@ def facets(self, lane):
         allowed_orders = library.enabled_facets(Facets.ORDER_FACET_GROUP_NAME)
         chosen_orders = self.orders or [default_order]
 
-        allowed_entrypoints = [
+        allowed_entrypoint_names = [
```

Comment:
This is just renaming the variable to make it clear that this list contains names, not EntryPoint objects.